app/views/videos/show/_video_frame.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="panel panel-default module">
+  <div class="panel-heading">
+    <b><%= video.title %></b>
+    <div id="toggle-star-<%= @video.id %>" class="pull-right">
+      <%= toggle_star_button(video) %>
+    </div>
+  </div>
+
+  <div class="h_iframe">
+      <img class="ratio" src="http://placehold.it/16x9" />
+      <iframe width="420" height="315" src="<%= video.youtube_link %>" frameborder="0" allowfullscreen></iframe>
+  </div>
+
+  <div class="video-footer">
+    <div class="progress pull-left">
```

Comment:
Let's remove the left progress bar for now. Progress for videos is represented as a % in the dashboard but maybe we can change it to stages? (Unassigned, In Translation, In Review, Completed). Worried that progress bar could be confused with likes on Youtube since they use a similar kind of progress bar.